{% extends "layout.html" %}

{% block title %}Engineering Success{% endblock %}
{% block lead %}
  <div class="page-header" id="engineering-header">
    <h1>Engineering Success</h1>
  </div>
{% endblock %}
{% block page_content %}

<style>
  figure {
    display: block;
    margin: auto;
    width: 50%;
    text-align: center;
  }
</style>

<div data-bs-spy="scroll" data-bs-target="#myScrollspy"  data-bs-offset="80" tabindex="0">
  <div class="container">
    <div class="row">
      <!-- Scrollspy Sidebar -->
      <nav class="col-12 col-md-3 col-lg-2" id="myScrollspy" aria-label="Page section navigation">
        <div class="sticky-lg-top scrollspy-offset">
          <nav class="nav nav-pills flex-column" role="navigation">
            <a class="nav-link" href="#summary" aria-current="false">Engineering Success</a>
            <a class="nav-link" href="#considerations" aria-current="false">Initial Considerations</a>
            <a class="nav-link" href="#cycles" aria-current="false">DBTL Cycles - Curli + MICP</a>
              <a class="nav-link ms-3" href="#cyc1" aria-current="false">DBTL Cycle 1</a>
              <a class="nav-link ms-3" href="#cyc2" aria-current="false">DBTL Cycle 2</a>
              <a class="nav-link ms-3" href="#cyc3" aria-current="false">DBTL Cycle 3</a>
              <a class="nav-link ms-3" href="#spcycle" aria-current="false">S. Pasteurii Cycle</a>
            <a class="nav-link" href="#references" aria-current="false">References</a>
          </nav>
        </div>
      </nav>
    
      <!-- Main Content -->
      <div class="col-12 col-md-9" id="bumper">
        <div id="summary" class="mb-5 row">
          <p>We conducted two parallel sets of DBTL cycles: </p>
          <ul class="mx-4">
            <li>We conducted research on bacteria engineered to produce curli fusion proteins, revealing potential oligomerization issues in curli fusion proteins.</li>
            <li>We successfully created biomineralized samples utilizing S. pasteurii’s natural biomineralization capabilities 
            </li>
          </ul>
        </div>
      <div id="considerations" class="mb-5 row">
        <h1>Initial Considerations</h1>
        <p>
          Our goal was to design bacteria that would simultaneously produce a sticky biofilm and induce biomineralization, capturing microplastics and forming an aggregate usable in construction. We chose to use curli fibers, a key component of E. coli biofilms, because they exhibit high adhesive properties and strong tensile mechanical properties (Zhang et al., 2017). Examining pathways involved in Microbially Induced Calcite Precipitation (MICP), we found that bacterial strains utilizing the urea hydrolysis pathway showed more calcite precipitation (~20–80%) in comparison with other metabolic pathways (Castro-Alonso et al., 2019). This pathway is primarily driven by the enzymes urease and carbonic anhydrase. To simultaneously produce curli fibers and induce biomineralization, we decided to emulate the fusion protein of curli fiber and amylase that Neel Joshi’s Biologically Fabricated Materials Lab designed (Birnbaum et al., 2021). Since carbonic anhydrase is a monomeric enzyme in the urea hydrolysis pathway, it was natural to design and experiment with a curli fiber and carbonic anhydrase fusion protein. We chose to use the strain E. coli K12 to express our fusion protein, as E. coli K12 can, under the right conditions, produce both curli fiber and carbonic anhydrase.             </p>
        <p>Our second goal was to create concrete biomineralized products. We chose to use S. pasteurii, as it naturally used the urea hydrolysis pathway, which eventually converts urea into ammonia and carbonate </p>
      </div>
      <hr/>

        <div id="cycles" class="mb-5">
          <h1>DBTL Cycles: Curli + MICP</h1>
          <h3>Overview of part: curli + sazCA</h3>
          <p>
            The goal of this part was to simultaneously produce curli fiber and induce biomineralization using one fusion protein. The <a href = "https://2023.igem.wiki/msp-maastricht/engineering.html">2023 iGEM team MSP-Maastricht’s</a> biomineralization carbonic anhydrase module inspired us to use sazCA, which is also the fastest carbonic anhydrase to date (Y. Zhu et al., 2021). As such, our final part’s primary goal was to produce a csgA-sazCA fusion protein.
          </p>
        </div>
        <div id="cyc1" class="mb-5">
          <h1>DBTL Cycle 1</h1>
          <h2>Design:</h2>
          <p>
            To simultaneously produce curli fiber and induce biomineralization, we designed a part [part link here] that could express a fusion protein of csgA, the primary curli monomer, and sazCA, the fastest carbonic anhydrase to date. We inserted our part—derived from the Neel Joshi Lab, 2023 iGEM team MSP-Maastricht and Zhu, Yinzhuang et al., 2021—into the Twist vector pET_blank (Amp), making our final plasmid [insert part here]. Hence, our transformed bacteria with this plasmid would have ampicillin resistance and would express our curli-sazCA fusion when iPTG is present.
          </p>
          <h2>Build:</h2>
          <p>
            Once our plasmid was synthesized by Twist, we performed chemical transformation on two tubes of wild E. coli K12, one of which we added our plasmid pET-csg-sazCA-amp. We recovered and plated the transformed cultures on both LB plates and LB/amp plates, letting the plates incubate overnight at 37C. 
            </p>
            <figure>
            <img 
              src="https://static.igem.wiki/teams/5761/results/khanlabbayarea-transformants.webp" 
              alt="Petri dishes showing LB/amp and LB plates with/without added DNA" 
              class="img-fluid rounded shadow">
            <figcaption>Top left: LB/amp plates, K12 without added DNA <br>
              Top right: LB/amp plate, K12 with added DNA<br> 
              Bottom left: LB plate, K12 without added DNA <br>
              Bottom right: LB plate, K12 with added DNA
            </figcaption>
          </figure>
          <p>
          <p>As hypothesized, for our bacteria without plasmid DNA, there was a smear on the LB plate and no bacteria on the LB/amp plate. For our bacteria with plasmid DNA, there was a smear on the LB plate and individual colonies on the LB/amp plate. This confirmed that our ampicillin was working properly, and that our E. coli K12 had likely been transformed. </p>
          <p>
            We further confirmed the transformation of our E. coli by performing colony PCR and sequencing via Plasmidsaurus. </p>
            <figure>
              <img 
                src="https://static.igem.wiki/teams/5761/results/khanlab-bayarea-colony-pcr-gel.webp" 
                alt="Colony PCR results" 
                class="img-fluid rounded shadow">
            <figcaption>
                From left to right, NEB 1kb+ ladder, NEB fastload 2 ladder, transformant culture 1, transformant culture 2, transformant culture 3.
            </figcaption>
            </figure>
            <p>
              The presence of dim bands around 4kb indicates the presence of our plasmid in all three transformant cultures.
              </p>
          <h2>Test:</h2>
          <p>
            Our first step was to test a variety of inducer concentration to find what optimized expression. To test expression  we utilized a congo red fluorescence assay inspired by (Kan et al., 2019) as well as a spin down curli assay from as well as a spin down congo red assay from iGEM team Delft-Leiden.
          </p>
          <figure>
            <img 
              src="https://static.igem.wiki/teams/5761/results/delta-a480-after-spin-down.svg"
              alt="Comparison of varying iPTG concentrations on absorbance after spin down of transformant culture. More negative absorbance indicates greater presence of Curli fibers. Graph taken from 8/17 curli absorbance assay."
              class="img-fluid rounded shadow">
              <figcaption>Comparison of varying iPTG concentrations on absorbance after spin down of transformant culture. More negative absorbance indicates greater presence of Curli fibers. Graph taken from 8/17 curli absorbance assay.</figcaption>
          </figure>
          <p>From these results, we decided to proceed with an inducer concentration of 0.2 mM.
          </p>
          <p>With our transformants, we then performed a congo red fluorescence assay to test for the presence of curli fibers. Congo red binds to amyloid fibrils like those found in curli fibers, producing more fluorescence at 525nm excitation and 625nm emission when bound to these fibrils. We induced each of our cultures with 0.2 mM iPTG and ran a congo red fluorescence assay. 
          </p>
          <figure>
            <img 
              src="https://static.igem.wiki/teams/5761/results/rna-extract-concentration-of-induced-vs-uninduced-samples.svg"
              alt="Graph of induced vs uninduced fluorescence readings of various cultures, with data in quadruplets. 1,2 and 3 are transformant cultures. Joshi is a curli expressing strain we received from the Joshi lab, intended as a positive control. Lb is a negative control of LB broth. "
              class="img-fluid rounded shadow">
              <figcaption>Graph of induced vs uninduced fluorescence readings of various cultures, with data in quadruplets. 1,2 and 3 are transformant cultures. Joshi is a curli expressing strain we received from the Joshi lab, intended as a positive control. Lb is a negative control of LB broth.</figcaption>
          </figure>
          <p>
            This assay supported our positive control expressing curli, but indicated to us that our transformants were struggling to express curli. Given that our transformants were confirmed by sequencing, we suspected that our protein had issues folding. 
            </p>
          <h2>Learn:</h2>
          <p>Using Alphafold 3, we modeled the oligomerization of our modified curli fiber as compared to our positive control, and saw significant differences in folding. </p>
          <figure>
            <img 
              src="https://static.igem.wiki/teams/5761/results/alphafold-generated-images.webp"
              alt="From left to right: Curli fiber monomers polymerizing normally, positive control Joshi lab strain curli fiber polymerization, our modified curli fiber polymerization. Note the pore like structure instead of the expected fiber formation."
              class="img-fluid rounded shadow">
              <figcaption>From left to right: Curli fiber monomers polymerizing normally, positive control Joshi lab strain curli fiber polymerization, our modified curli fiber polymerization. Note the pore like structure instead of the expected fiber formation. </figcaption>
          </figure>
        </div>
        <hr/>
        <div id="cyc2">
          <h1>DBTL Cycle 2</h1>
          <h2>Design:</h2>
          <p>Having a suggestion from the Dry Lab, we designed a Wet Lab based experiment to confirm if issues were happening at the protein folding stage or later. </p>
          <p>To do this we designed individual IPTG inducible constructs with our proteins, Curli and Carbonic anhydrase. Our goal was to confirm that expression of each individual protein was successful, which would support the conclusion that the problem was unique to the modified protein. </p>
          <div class="image-container">
            <figure>
              <img 
                src="https://static.igem.wiki/teams/5761/results/pet-csg-h6-amp-map.webp"
                alt="Our two new constructs, IPTG inducible curli and carbonic anhydrase."
                class="img-fluid rounded shadow">
            </figure>
            <figure>
              <img 
                src="https://static.igem.wiki/teams/5761/results/pet-sazca-amp-map.webp"
                alt="Our two new constructs, IPTG inducible curli and carbonic anhydrase."
                class="img-fluid rounded shadow">
                <figcaption>Our two new constructs, IPTG inducible curli and carbonic anhydrase. 
                </figcaption>
            </figure>
          </div>
          <p>
            We additionally planned a RT-PCR assay. RT-PCR would allow us to view the amount of mRNA produced from a given construct, via first converting it to cDNA and then amplifying it. This would potentially confirm that transcription was present in our modified construct, and support that the issue lied at the translation/protein folding stage. 
            </p>
            <p>In designing RT-PCR assay, we included a positive control of the RSS housekeeping gene in E.coli to confirm the validity of our assay.</p>
            <h2>Build:</h2>
            <p>We utilized our previously made competent cells to create transformants of both constructs. 
              We ordered reagents required to run a RT-PCR assay, including custom primers and Reverse Transcriptase. 
              </p>
            <h2>Test:</h2>
            <p>We ran a congo red fluorescence assay on both our Curli only construct, confirming curli individually did express successfully in E.coli K-12 and that the problem resided in the fusion protein.</p>
            <figure>
              <img 
                src="https://static.igem.wiki/teams/5761/results/induced-vs-uninduced-culture-fluorescence-curli-construct.svg"
                alt="Induced vs. uninduced culture flourescence curli construct"
                class="img-fluid rounded shadow">
            </figure>
            <p>The RT PCR gel, see below, showed mrna expression was present, indicating the issue had to do with protein folding further down the line. </p>
            <figure>
              <img 
                src="https://static.igem.wiki/teams/5761/results/rt-pcr-gel-results.webp"
                alt="Gel of RT results. Ladder, K12, Joshi, Joshi induced, Curli, Curli induced, Curli-CA, Curli-CA induced."
                class="img-fluid rounded shadow">
                <figcaption>
                  Gel of RT results. Ladder, K12, Joshi, Joshi induced, Curli, Curli induced, Curli-CA, Curli-CA induced.
                  </figcaption>
            </figure>
            <p>We additionally confirmed that expression of Curli fibers functioned within E.coli K-12 when not modified, through the creation and characterization of a construct involving solely Curli fibers. We used the fluorescence assay described earlier to demonstrate presence of protein expression.</p>
            <p>Finally, we also tested expression of Carbonic anhydrase through the Wilbur Anderson assay <a href = "https://parts.igem.org/Part:BBa_K4665005">2023 iGEM team MSP-Maastricht</a>. Unfortunately, we ran into issues with continuously reducing measurements from when our CO2 solution was opened, a known issue with the assay. In the limited time we had remaining, we were unable to further optimize the assay to see usable results. </p>
            <h2>Learn:</h2>
            <p>Through both expression of unmodified proteins and RT-PCR data, we were able to support our hypothesis that the issue with our modified construct occurs at the protein folding/oligomerization stage.</p>
            <h2>Overview of Part: Curli + Urease</h2>
            <p>As soon as we discovered that there may be potential folding issues with our curli fiber and carbonic anhydrase fusion protein, we switched tracks to design a part with urease instead of carbonic anhydrase. Unfortunately, due to time constraints, we were unable to make it past the design and build stage. </p>
        </div>

        <div id="cyc3">
          <h1>DBTL Cycle 3: csg + Urease</h1>
          <h2>Design:</h2>
          <p>As soon as we discovered that there may be potential folding issues with our curli fiber and carbonic anhydrase fusion protein, we switched tracks to design a part with urease instead of carbonic anhydrase. Unfortunately, due to time constraints, we were unable to make it past the design and build stage. </p>
          <figure style = "width: 100%; height: auto;">
            <img 
              src="https://static.igem.wiki/teams/5761/results/laci-csg-ure-v2-map.webp"
              alt="Laci csg urease v2 map"
              class="img-fluid rounded shadow">
          </figure>
          <p>This construct attached the urease gene cluster to the sequence after the curli fiber gene cluster. The urease gene cluster was taken from part BBa_K5205014, designed by the 2024 Hangzhou-SDG iGEM team. We planned to create this construct by inserting the urease gene cluster into one of our existing curli plasmids. </p>
          <p>As we received a grant from Ansa Biotechnology to synthesize a plasmid with less than 10kb of insert, we ended up designing a subsequent construct that used arabinose instead of iPTG to induce expression. </p>
          <figure style = "width: 100%; height: auto;">
          <img 
            src="https://static.igem.wiki/teams/5761/results/arabad-csg-ure-v2-map.webp"
            alt="Arabad csg urease v2 map"
            class="img-fluid rounded shadow">
        </figure>
        <h2>Build:</h2>
        <p>We ordered the urease gene cluster in two gene fragments synthesized by IDT, with the plan to ligate them via NEB HiFi DNA Assembly and insert into our plasmid pET_csg_Amp via restriction enzyme digest. At the end of September, we also received our construct from Ansa Biotechnology. Once we had transformed bacteria, we planned to assess curli fiber presence via congo red fluorescence assays and urease activity with a phenol red pH assay (protocol:).</p>
        <p>Unfortunately, due to time constraints and our focus on characterizing csg-sazCA, we were unable to transform and perform experiments on this particular construct. </p>
        </div>
        <div id="spcycle">
          <h1>DBTL Cycles - S. pasteurii</h1>
          <h2>Overview: S. pasteurii</h2>
          <p>In parallel with our main experimental DBTL cycles, we conducted experiments to create a rock-like material and provide a proof of concept. We chose to work with S. pasteurii, a naturally biomineralizing bacteria, as it naturally utilized the urea hydrolysis pathway. We procured S. pasteurii and the necessary protocol from Kelsey De Frates, from the Hernandez Lab at UCSF. </p>
          <h1>DTBL Cycle 1:</h1>
          <h2>Design:</h2>
          <p>To create a biomineralized hard product that could be tested, we designed a mold following ASTM standards for tensile/compressive strength testing with scaled down dimensions. The resulting product had a cross-sectional area of 2 squared, allowing us to later estimate tensile strength.
          </p>
          <figure>
            <img 
              src="https://static.igem.wiki/teams/5761/results/cad-of-mold.webp"
              alt="Image of inverse mold that was created in CAD, then 3d printed in PETG. The mold was then used to cast PDMS which after curing became our mold for biomineralization."
              class="img-fluid rounded shadow">
              <figcaption>Image of inverse mold that was created in CAD, then 3d printed in PETG. The mold was then used to cast PDMS which after curing became our mold for biomineralization.</figcaption>
          </figure>
          <h2>Build:</h2>
          <p>We then built a PDMS mold from a 3D printed PETG inverse based on our CAD model, following our protocol “Manufacturing PDMS Channel-Pore Microfluidic Devices.”  
          </p>
          <figure>
            <img
              src="https://static.igem.wiki/teams/5761/results/pdms-mold-next-to-petg-inverse.webp"
              alt="ann orange block with an oar shape and a clear block of the same shape but smaller"
              class="img-fluid rounded shadow">
            <figcaption>
              Image of PDMS casted from a 3d printed PETG inverse. The PDMS mold is then used to create a biomineralized sample in the appropriate shape for strength testing. 
            </figcaption>
          </figure>
          <p>In parallel, we cultured S. pasteurii from glycerol stocks at 30C. Having acquired the appropriate mold, media, and S. pasteurii culture, we began biomineralization, following protocol from Troyer et al. Our three molds, one which was our PDMS mold and the other two which were normal 3D molds, were filled with sand to act as nucleation sites. For one month, we conducted daily interchange between S. pasteurii culture and precipitation media to induce calcite precipitation. </p>
          <h2>Test:</h2>
          <p>Samples were then taken out of molds and sterilized as per our sterilization protocol [link to experiment page, sterilization protocol]. We were able to confirm that our protocol successfully sterilized our samples via a swab test. While the process of removing the samples from our molds broke the mineralized material into multiple pieces rendering us unable to measure strength, we can see that the sand has clearly turned into a much more coarse material. </p>
          <figure>
            <img
              src="https://static.igem.wiki/teams/5761/results/khanlabbayarea-sterilized-samples.webp"
              alt="three containers with minerals decreasing in size"
              class="img-fluid rounded shadow">
            <figcaption>
              Image of our sterilized samples. Note that the right most sample, prepared in a PDMS mold, performed the best, breaking into 3 clean pieces on removal.
            </figcaption>
          </figure>
          <h2>Learn:</h2>
          <p>From this process, we learned that fine-grain sand worked best for biomineralization. 
            [Make note of S. pasteurii slow growth + contamination issues we had here, as well as sterilization techniques and practices we learned over time.] </p>
        </div>
        <div id="references">
          <h1>References</h1>
          <p class="apa-reference">
            Birnbaum, D. P., Manjula‐Basavanna, A., Kan, A., Tardy, B. L., & Joshi, N. S. (2021). Hybrid living capsules autonomously produced by engineered bacteria. <em> Advanced Science</em>. 8(11). https://doi.org/10.1002/advs.202004699
          </p>
          <p class="apa-reference">
            Castro-Alonso, M. J., Montañez-Hernandez, L. E., Sanchez-Muñoz, M. A., Franco, M. R. M., Narayanasamy, R., & Balagurusamy, N. (2019). Microbially Induced calcium carbonate precipitation (MICP) and its potential in bioconcrete: Microbiological and Molecular concepts. <em>Frontiers in Materials, 6</em>. https://doi.org/10.3389/fmats.2019.00126 
          </p>
          <p class="apa-reference">
            Kan, A., Birnbaum, D. P., Praveschotinunt, P., & Joshi, N. S. (2019). Congo red fluorescence for rapid in situ characterization of synthetic curli systems. <em>Applied and Environmental Microbiology, 85</em>(13), e00434-19. https://doi.org/10.1128/AEM.00434-19 
          </p>
          <p class="apa-reference">
            Zhang, Y., Wang, A., DeBenedictis, E. P., & Keten, S. (2017). Bending energy penalty enhances the adhesive strength of functional amyloid curli to surfaces. <em>Nanotechnology, 28</em>(46), 464002. https://doi.org/10.1088/1361-6528/aa8f72
          </p>
          <p class="apa-reference">
            Zhu, Y., Liu, Y., Ai, M., & Jia, X. (2021). Surface display of carbonic anhydrase on Escherichia coli for CO2 capture and mineralization. <em>Synthetic and Systems Biotechnology, 7</em>(1), 460–473. https://doi.org/10.1016/j.synbio.2021.11.008
          </p>
          <p class="apa-reference">
            Troyer, E., Berninghaus, A., Gerlach, R., Foreman, C., Joyce, J., West, C., and A. J. Phillips. "Biomineralized Art: Using Microbes and Minds to Make Mountains." Paper presented at the 51st U.S. Rock Mechanics/Geomechanics Symposium, San Francisco, California, USA, June 2017.
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
{% endblock %}
